### PR TITLE
feat(AWS SQS): Support `Properties.ScalingConfig.MaximumConcurrency`

### PR DESCRIPTION
**Pull request by jasonrowsell**
Closes: #11676

### CODE REVIEW

**Review by medikoo — COMMENTED** [review]
@jasonrowsell that looks very good! I have just one minor style suggestion

File: lib/plugins/aws/package/compile/events/sqs.js [hunk 1]
```diff
@@ -111,6 +112,12 @@ class AwsCompileSQSEvents {
               };
             }
 
+            if (event.sqs.maximumConcurrency != null) {
```
[the remark this answers is not in the document]
Comment (jasonrowsell) [reply lib/plugins/aws/package/compile/events/sqs.js]: Sorry didn't see this, updated :) @medikoo

**Comment by jasonrowsell** [comment]
@medikoo Been following the other related PRs and this comment https://github.com/serverless/serverless/pull/11679#pullrequestreview-1250318531. Any objections to getting this merged?

**Review by jeanphilippelingrand — APPROVED** [review]

**Review by medikoo — APPROVED** [review]
Thank you @jasonrowsell !

**Pull request merged**